### PR TITLE
Series api client

### PR DESCRIPTION
**Pull request by flipper83**
- add api for work with series.

### CODE REVIEW

**Comment by flipper83** [comment]
@Serchinastico @pedrovgs I need this branch for close the rosie demo ^_^. thank you guys.

File: MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SerieApiClientTest.java [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+  }
+
+  private List<Integer> getAnyEvents() {
+    List<Integer> events = new ArrayList<>();
+    events.add(ANY_EVENT_1);
+    events.add(ANY_EVENT_2);
+    return events;
+  }
+
+  private SerieApiClient givenSerieApiClient() {
```
Comment (pedrovgs) [inline MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SerieApiClientTest.java]: In english, Serie in singular is written as SerieS. Please review all the serie you have as local variable and also the json file.

File: MarvelApiClient/src/main/java/com/karumi/marvelapiclient/SerieApiClient.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+  public MarvelResponse<SeriesDto> getAll(SeriesQuery seriesQuery) throws MarvelApiException {
+    SerieApiRest api = getApi(SerieApiRest.class);
+
+    Map<String, Object> queryAsMap = seriesQuery.toMap();
+    Call<MarvelResponse<SeriesDto>> call = api.getSeries(queryAsMap);
+    return execute(call);
+  }
+
+  public MarvelResponse<SerieDto> getSerie(String serieId) throws MarvelApiException {
```
Comment (pedrovgs) [inline MarvelApiClient/src/main/java/com/karumi/marvelapiclient/SerieApiClient.java]: We can rename this to getSeriesById :)

**Comment by pedrovgs** [comment]
Review the typo and we can merge this feature :) :+1:

File: MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SerieApiClientTest.java (hunk 1 above)
Comment (flipper83) [inline MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SerieApiClientTest.java]: fuck, this is going to be a hell :S, ok, I must rename some classes.

**Comment by flipper83** [comment]
the typo is going to be big one, but ok, it's faster.

**Pull request merged**